api/errors.go

```diff
@@ -0,0 +1,14 @@
+package api
+
+import "fmt"
+
+// Error is the error type returned when API requests result in an error.
+type Error struct {
+	Endpoint string
```

Comment:
Could the fields be commented, eg. making clear that `Code` is the HTTP status code coming back from the MyRadio response but `Payload` is (I guess) the raw description string coming out of MyRadio?